adal/src/main/java/com/microsoft/aad/adal/WebFingerMetadataRequestor.java

```diff
@@ -99,6 +99,7 @@ WebFingerMetadata parseMetadata(final HttpWebResponse webResponse) throws Authen
      * @return the URL of the WebFinger document
      * @throws MalformedURLException if the URL could not be constructed
      */
+    @SuppressWarnings("PMD")
```

Comment:
Do you know what kind of PMD issue reported here?

Reply:
InsufficientStringBufferDeclaration, it's quite trival. 
http://pmd.sourceforge.net/pmd-4.3.0/rules/strings.html ,
 search InsufficientStringBufferDeclaration : 

Failing to pre-size a StringBuffer properly could cause it to re-size many times during runtime. This rule checks the characters that are actually passed into StringBuffer.append(), but represents a best guess "worst case" scenario. An empty StringBuffer constructor initializes the object to 16 characters. This default is assumed if the length of the constructor can not be determined.

This rule is defined by the following Java class: net.sourceforge.pmd.rules.strings.InsufficientStringBufferDeclaration

Example:

                
```
public class Foo {
    void bar() {
        StringBuffer bad = new StringBuffer();
        bad.append("This is a long string, will exceed the default 16 characters");//bad
        StringBuffer good = new StringBuffer(41);
        good.append("This is a long string, which is pre-sized");//good
    }
}
```